### PR TITLE
RU & UA : Cleaning + namefix

### PR DESCRIPTION
**Pull request by BellezaEmporium**
RU :
- Adaptive links when possible.
- Correcting certain links that weren't working on fixing time.
- Give alternative links for certain streams.
- Cleaning dupe links or links being in adaptive.
- Correct naming (per discussion with @Dum4G , spotting a naming anomaly).

UA : 
- Adaptive links when possible
- Give alternative links for certain streams
- Cleaning dupe links or links being in adaptive
- Moving certain links to a more secure version of their server.

### CODE REVIEW

File: channels/ru.m3u [hunk 1]
```diff
@@ -369,16 +283,10 @@ https://strm.yandex.ru/kal/diving/diving0.m3u8
 http://strm.yandex.ru/kal/dialogi/dialogi0.m3u8
 #EXTINF:-1 tvg-id="Dozhd.ru" tvg-name="Дождь" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",Дождь [No audio]
 https://a3569457435-s78511.cdn.ngenix.net/open/TVRain_noaudio/playlist_DVR.m3u8
+#EXTINF:-1 tvg-id="Dozhd.ru" tvg-name="Дождь" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",Дождь [No audio]
```
Comment (Dum4G) [inline channels/ru.m3u:286]: has audio

File: channels/ru.m3u [hunk 2]
```diff
@@ -502,18 +412,12 @@ http://hls-rossp.cdnvideo.ru/hls/5ace67f0dc96bf0a90d8a2b7/playlist.m3u8
 http://hls-rossp.cdnvideo.ru/hls/5ace6814e563f22e4ddc1f44/playlist.m3u8
 #EXTINF:-1 tvg-id="LitKlubTVStihi.ru" tvg-name="ЛитКлуб ТВ - Стихи" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",ЛитКлуб ТВ - Стихи
 http://hls-rossp.cdnvideo.ru/hls/5aac88ffdc96bf05a7f78899/playlist.m3u8
-#EXTINF:-1 tvg-id="LitKlubTVProza.ru" tvg-name="ЛитКлуб ТВ Проза" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",ЛитКлуб ТВ Проза (720p)
-http://video.platformcraft.ru/hls/rossp/proza720/playlist.m3u8
```
Comment (Dum4G) [inline channels/ru.m3u:506]: works

File: channels/ru.m3u [hunk 3]
```diff
@@ -502,18 +412,12 @@ http://hls-rossp.cdnvideo.ru/hls/5ace67f0dc96bf0a90d8a2b7/playlist.m3u8
...
-#EXTINF:-1 tvg-id="LitKlubTVSlovo.ru" tvg-name="ЛитКлуб ТВ Слово" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",ЛитКлуб ТВ Слово (720p)
-http://video.platformcraft.ru/hls/rossp/slovo720/playlist.m3u8
```
Comment (Dum4G) [inline channels/ru.m3u:508]: works

File: channels/ru.m3u [hunk 4]
```diff
@@ -502,18 +412,12 @@ http://hls-rossp.cdnvideo.ru/hls/5ace67f0dc96bf0a90d8a2b7/playlist.m3u8
...
-#EXTINF:-1 tvg-id="LitKlubTVStihi.ru" tvg-name="ЛитКлуб ТВ Стихи" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",ЛитКлуб ТВ Стихи (720p)
-http://video.platformcraft.ru/hls/rossp/stihi720/playlist.m3u8
```
Comment (Dum4G) [inline channels/ru.m3u:510]: Works

File: channels/ru.m3u [hunk 5]
```diff
@@ -664,18 +560,16 @@ http://ott-cdn.ucom.am/s17/index.m3u8
 #EXTINF:-1 tvg-id="NTVStil.ru" tvg-name="НТВ Стиль" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",НТВ Стиль (480p)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148
 http://ott-cdn.ucom.am/s77/index.m3u8
-#EXTINF:-1 tvg-id="NTVPlusSport.ru" tvg-name="НТВ+ Спорт" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",НТВ+ Спорт (720p)
+#EXTINF:-1 tvg-id="1Plus1Sport.ru" tvg-name="1+1 Спорт" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",1+1 Спорт (720p)
```
Comment (Dum4G) [inline channels/ru.m3u:563]: UA channel

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/ru.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/ru.m3u:506]: Redirects to the Rossiya link above.

File: channels/ru.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/ru.m3u:286]: Copied and pasted without getting the mention off indeed. Gonna change that.

File: channels/ru.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/ru.m3u:508]: Same case

File: channels/ru.m3u (hunk 4 above)
Comment (LaneSh4d0w) [reply channels/ru.m3u:510]: Same case

File: channels/ru.m3u (hunk 5 above)
Comment (LaneSh4d0w) [reply channels/ru.m3u:563]: Gonna move that.

File: channels/ru.m3u (hunk 2 above)
Comment (Dum4G) [reply channels/ru.m3u:506]: Which one? Those three channels are all about different kinds of poetry and I got some differnt streams

Comment (LaneSh4d0w) [reply channels/ru.m3u:506]: All links that were removed redirects to their Rossiya variant

(for example, if you use Firefox, and use the Network sniffer, you'll see a redirection going to the hls link above)

Comment (Dum4G) [reply channels/ru.m3u:506]: Would you be kind to show me the final link, please? I don't see any other LitClubs in the playlist

Comment (LaneSh4d0w) [reply channels/ru.m3u:506]: image

Comment (Dum4G) [reply channels/ru.m3u:506]: Ah yes, my bad. Tho I'd keep both links in case those hashes gonna change over time

Comment (jkemp814) [reply channels/ru.m3u:506]: That network sniffer is really powerful. I see you got that extension to work. Really makes it convenient. Another extension I use is Clear URL. It strips all those requests about your browser before sending to server. Most of the URL now-a-days are spying using the URL. That's how they know you are not using their app, but instead using a browser.

Comment (LaneSh4d0w) [reply channels/ru.m3u:506]: @Dum4G Well, I don't know if it's useful to keep the other link if it just redirects to the source link we also have. Better change those links directly and making sure we have no redirections whatsoever.

File: channels/ua.m3u [hunk 6]
```diff
@@ -1,10 +1,8 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="1Plus1.ua" tvg-name="1+1" tvg-country="UA" tvg-language="Ukrainian" tvg-logo="" group-title="",1+1
-http://spacetv.in/stream/DIPK8H19CP8/172.m3u8
+#EXTINF:-1 tvg-id="1Plus1Sport.ru" tvg-name="1+1 Спорт" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",1+1 Спорт (720p)
```
Comment (Dum4G) [inline channels/ua.m3u:2]: tvg-country="UA" tvg-language="Ukrainian" I guess

File: channels/ua.m3u [hunk 7]
```diff
@@ -45,13 +35,7 @@ https://stream.uagit.tv/gittv.m3u8
 http://cdn1.live-tv.od.ua:8081/a1od/gtvod-abr/a1od/gtvod-480p/playlist.m3u8
 #EXTINF:-1 tvg-id="GTV.ua" tvg-name="GTV" tvg-country="UA" tvg-language="Ukrainian" tvg-logo="https://i.imgur.com/Rc6UGkb.jpg" group-title="",GTV (720p)
 http://cdn1.live-tv.od.ua:8081/a1od/gtvod-abr/a1od/gtvod-720p/playlist.m3u8
-#EXTINF:-1 tvg-id="HDFashionLifestyle.ua" tvg-name="HD Fashion & Lifestyle" tvg-country="UA" tvg-language="" tvg-logo="" group-title="Lifestyle",HD Fashion & Lifestyle
-http://95.67.47.114/hls/hdfashion_ua_hi/index.m3u8
-#EXTINF:-1 tvg-id="HDFashionLifestyleTV.ua" tvg-name="HD Fashion & Lifestyle TV" tvg-country="UA" tvg-language="" tvg-logo="" group-title="Lifestyle",HD Fashion & Lifestyle TV
-http://95.67.47.114/hls/hdfashion_ua_mid/index.m3u8
```
Comment (Dum4G) [inline channels/ua.m3u:51]: also works, why not to keep it then

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/ua.m3u (hunk 7 above)
Comment (LaneSh4d0w) [reply channels/ua.m3u:51]: #EXTINF:-1 tvg-id="HDFashionLifeStyle.ua" tvg-name="HDFashion & LifeStyle" tvg-country="UA" tvg-language="Ukrainian" tvg-logo="https://i.imgur.com/LtQL1q9.jpg" group-title="Lifestyle",HDFashion & LifeStyle
http://95.67.47.115/hls/hdfashion_ua_mid/index.m3u8

Hmm... wonder why they got 2 IPs streaming the same content...

Comment (LaneSh4d0w) [reply channels/ua.m3u:51]: Rectified.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**